### ISSUE TITLE
#-699309264------------- Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1462]
<strong>Source:</strong>`<mark class="label modifier-class c-label--yellow" data-name="statusCount1">17</mark>`

<strong>Element Location: </strong>.c-label--yellow

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix any of the following:
  Element has insufficient color contrast of 2.46 (foreground color: #ffffff, background color: #e8912e, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

<strong>Tags:</strong> cat.color,wcag2aa,wcag143,TTv5,TT13.c,EN-301-549,EN-9.1.4.3,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=axeAPI
